### DISCUSSION TITLE
Deploy ActorSystem as standalone service

### DISCUSSION THREAD

**Discussion by mrxrsd** [akkadotnet/akka.net#4886] [Q&A]
Hi,

In all my cases I've had been deployed my actorsystem inside my .net core hosted app. But this scenario restrict me because I can't scale out my .net core pods. Am I right?

How can I deploy a actorsystem as standalone service? Is it using Akka Cluster/Remote ?

**Comment by ismaelhamed** [comment]
You can pretty much host your application in any .NET process (console app, web/generic hosts, WCF, etc.), and still have the ability to form a cluster and scale. 

All the clustering features are included in the `Akka.Cluster*` modules: https://getakka.net/articles/clustering/cluster-overview.html

**Comment by Aaronontheweb** [comment]
I made a YouTube video about some of this 1-2 weeks ago:

https://petabridge.com/blog/akkadotnet-ihostedservice/
https://www.youtube.com/watch?v=gpaaCBtKLss

**Reply by mrxrsd** [reply]
Thanks Aaron! I will take a look.

**Reply by Aaronontheweb** [reply]
No problem! Basically, you're just using a console process that runs Akka.NET in the background as an `IHostedService` - you can scale that horizontally up and down by just adding / removing nodes to your cluster.
